test/pretty.test.js

```diff
@@ -354,7 +354,7 @@ test('works without time', function (t) {
   var prettier = pretty()
   prettier.pipe(split(function (line) {
     t.ok(line.match(/.*hello world$/), 'end of line matches')
-    t.ok(line.match(/^INFO.*/), 'includes level')
+    t.ok(line.match(/(?!^)INFO.*/), 'includes level')
```

Comment:
why this change?

Reply:
my bad, I finally figure out why previouse code doing this
```js
    line = (levelFirst)
        ? asColoredLevel(value) + ' ' + formatTime(value)
        : formatTime(value, ' ') + asColoredLevel(value)
```
I change levelFirst code  into
```js
    line = (levelFirst)
        ? asColoredLevel(value) + ' ' + formatTime(value)
        : formatTime(value) + ' ' + asColoredLevel(value)
```
cause the start of output is `a space + level`, then missing time test will fail
i will make this to previouse code

Reply:
I remove the previouse test I add, because the default output is `asISODate`, I do not have to add this test again. 
I need some help, how to add test for `asLocalISODate`, it only expose to CLI use. Separate code into js? expose method from pretty.js?

Reply:
Simplest (being a relative term) would be to use `child_process.fork` and parse the `stdout` stream of the child.